src/petals/__init__.py

```diff
@@ -17,13 +17,11 @@
 from petals.utils import *
 from petals.utils.logging import initialize_logs as _initialize_logs
 
-__version__ = "2.3.0.dev1"
+__version__ = "2.3.0.dev2"
 
 
 if not os.getenv("PETALS_IGNORE_DEPENDENCY_VERSION"):
-    assert (
-        version.parse("4.32.0") <= version.parse(transformers.__version__) < version.parse("4.35.0")
-    ), "Please install a proper transformers version: pip install transformers>=4.32.0,<4.35.0"
+    version.parse(transformers.__version__) == version.parse("4.37.1")
```

Comment:
Nit: we can safely support up to 4.38.0 (not including that). Can we make it

4.37.1 <= version < 4.38.0 ?